api_server.py

```
@@ -5,7 +5,7 @@
 import os
 import sqlite3
 from datetime import datetime, timedelta, timezone
-from typing import Dict, List, Optional, Tuple
+from typing import Dict, List, Optional, Tuple, Union, cast
```

Ruff (0.15.7)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)